project2/index.html

```diff
@@ -5,20 +5,49 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
     <title>Watch National Treasure</title>
 </head>
 
 <body>
-    <div class="box box-1">Directed and produced by John Turteltaub! Rotten Tomatoes rates it 45%.</div>
-    <figure>
-        <img src="nick-cage-in-national-treasure-promo.jpg" alt="Nick Cage in National Treasure">
-        <figcaption>Nick Cage stars in National Treasure</figcaption>
-    </figure>
-    <div class="boxed">
-        Nick Cage stars in National Treasure.
+    <div class="card">
+        <h1>National Treasure</h1>
+        <h2>"action-packed" - Risa Benedyk</h2>
+        <h3>Rotten Tomatoes gives it 45%.</h3>
+        <h4>Directed and produced by John Turteltaub!</h4>
+        <h5>RATED PG-13</h5>
+        <p>Nicolas Cage plays Benjamin Franklin Gates, a historian and amateur cryptologist searching for a
+            lost treasure of precious metals, jewelry, artwork and other artifacts that was accumulated into a
+            single massive stockpile by looters and warriors over many millennia starting in Ancient Egypt, later
+            rediscovered by warriors who form themselves into the Knights Templar to protect the treasure,
+            eventually hidden by American Freemasons during the American Revolutionary War. A coded map on the back
+            of the Declaration of Independence points to the location of the "national treasure," but Gates is not
+            alone in his quest. Whoever can steal the Declaration and decode it first will find the greatest treasure
+            in history. </p>
+        <figure>
+            <img src="images/nick-cage-in-national-treasure-promo.jpg" alt="Nick Cage in National Treasure">
```

Comment:
Remember to optimize your images; this one is 1280px wide, but your card can only be 640px wide, so you're loading a much larger image than is necessary for the users.

1. Resize image using local tools (PhotoShop or even image preview on macOS).
1. Optimize via [tinypng](https://tinypng.com/) (works for JPGs, not just PNGs 👍🏻)